app/views/pace/online.html.erb

```diff
@@ -1,3 +1,5 @@
+<% set_meta_tags(refresh: @redirect_string) %>
```

Comment:
I'm finding that this appends the redirect string to the existing URL rather than replaces it when I run it locally. I'm not sure if that is only a problem related to the environment.